src/shared/components/mutationMapper/MutationMapper.tsx

```diff
@@ -487,14 +494,12 @@ export default class MutationMapper<P extends IMutationMapperProps> extends Reac
     @action
     protected open3dPanel() {
         this.trackVisibility[TrackNames.PDB] = 'visible';
-        this.props.store.pdbChainDataStore.selectFirstChain();
```

Comment:
For simplicity, this logic is now moved into `ProteinChainPanel`, we only set the visibility here.